src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -673,8 +756,8 @@ public String addToInstancePublicIp(Infrastructure infrastructure, String instan
         ResourceGroup resourceGroup = azureService.resourceGroups().getByName(vm.resourceGroupName());
 
         // Try to retrieve the desired public IP address or create a new one
-        PublicIpAddress publicIpAddress = Optional.ofNullable(optionalDesiredIp)
-                                                  .map(opt -> azureService.publicIpAddresses()
+        PublicIPAddress PublicIPAddress = Optional.ofNullable(optionalDesiredIp)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)